spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -421,17 +421,17 @@ function pendingInflightPacketSequences(
 function isAuthorizedUpgrader(address: string): boolean
 ```
 
-`getUpgradeTimeout` will return the upgrade timeout specified for the given channel. This may be a chain-wide parameter, or it can be a parameter chosen per channel. This is an implementation-level detail, so only the function signature is specified here. Note this should retrieve some stored timeout delta for the channel and add it to the current height and time to get the absolute timeout values.
+`getTimeout` will return the upgrade timeout specified for the given channel. This may be a chain-wide parameter, or it can be a parameter chosen per channel. This is an implementation-level detail, so only the function signature is specified here. Note this should retrieve some stored timeout delta for the channel and add it to the current height and time to get the absolute timeout values.
 
 ```typescript
-// getUpgradeTimeout
-function getUpgradeTimeout(portIdentifier: string, channelIdentifier: string) UpgradeTimeout {
+// getTimeout
+function getTimeout(portIdentifier: string, channelIdentifier: string) Timeout {
```

Comment:
And the return type here is also good. But Every other change I would revert, since it makes sense still to have the word `upgrade` in the variable and functions names.

Reply:
@crodriguezvega I have reverted the changes